emcc.py

```diff
@@ -2092,7 +2092,7 @@ def do_minify(): # minifies the code. this is also when we do certain optimizati
         # BINARYEN_METHOD with something that doesn't use the polyfill, then we don't need it.
         if not shared.Settings.BINARYEN_METHOD or 'interpret' in shared.Settings.BINARYEN_METHOD:
           logging.debug('integrating wasm.js polyfill interpreter')
-          wasm_js = open(os.path.join(binaryen_bin, 'wasm.js')).read()
+          wasm_js = open(os.path.join(shared.Settings.BINARYEN_ROOT, 'share', 'binaryen', 'wasm.js')).read()
```

Comment:
I'll remove it from this PR, and we'll fix it more intentionally in a separate one